src/controller/event/handler/init.go

```diff
@@ -21,47 +21,47 @@ import (
 
 func init() {
 	// notification
```

Comment:
how about wrap a function to handle like `checkErr(notifier.Subscribe())`?

```go
func checkError(err error) {
    if err != nil {
        log.Error(err)
    }
}
```

Reply:
> how about wrap a function to handle like `checkErr(notifier.Subscribe())`?
> 
> ```go
> func checkError(err error) {
>     if err != nil {
>         log.Error(err)
>     }
> }
> ```

if you don't mind, I would prefer to ignoring the error returned by calling `notifier.Subscribe()`

Reply:
will the notifier.Subscribe() have the error in fact? if no error will be happened how about refactor this function to not return error.